<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCities</class>
 <widget class="QMainWindow" name="ShowCities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>174</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vis innlagte byer og steder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>jobber.png</normaloff>jobber.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCity">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>89</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nytt stedsnavn*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Slett</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountryName">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>123</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>$countryName</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>528</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Her kan du se stedene som er lagt inn i databasen. Vil du bytte ID-en til et sted, må dette gjøres via kommandolinja. Alle felt må fylles ut.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>55</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nåværende stedsnavn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagre</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCountryID">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>123</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LandID*</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCityID">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>55</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCityName">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>89</y>
      <width>159</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCountryID">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>123</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFirst">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPrev">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLast">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>157</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
